### ISSUE TITLE
[Accessibility] Narrator cannot focus on and announce the list items when pressing up/down keyboard for ListView/ListBox/CheckListBox

### ISSUE THREAD

**Issue by Amy-Li03** [dotnet/winforms#10259]
### .NET version

.NET 9.0.100-alpha.1.23558.10

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

It's a regression issue, doesn't repro on .NET 8.0/7.0/6.0.

### Issue description

Narrator cannot focus on and announce the list items when pressing up/down keyboard for ListView/ListBox/CheckListBox.

**The behavior for .NET 9.0:**
![9 0](https://github.com/dotnet/winforms/assets/62929087/4d70623b-6503-48bb-91d6-e2f6ff479540)
![image](https://github.com/dotnet/winforms/assets/62929087/9432a837-3e89-4405-9c1d-670baeb188af)

**The behavior for .NET 8.0:**
![8 0](https://github.com/dotnet/winforms/assets/62929087/03f0f719-4c25-480f-8c4d-5722ec12d07f)

### Steps to reproduce
[WinFormsApp134.zip](https://github.com/dotnet/winforms/files/13304574/WinFormsApp134.zip)

1. Create a WinForms .NET Core application with ListView control.
2. Add some columns and Items for it.
3. Set ListView's View as Details.
4. Build and run the application.
5. Launch the Narrator tool, try to focus on the first item of ListView.
6. Press up/down keyboard to focus on the next item.

**Comment by Eudora-Li01** [comment]
verified this issue on .NET 9.0+latest dll files, it has been fixed. narrator can focus on and announce the list items when pressing up/down keyboard for ListView/ListBox/CheckListBox controls.

https://github.com/dotnet/winforms/assets/130345015/c04c185f-e0d0-4aba-8c4f-2c8b0d8c237a
